docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -66,12 +59,6 @@ you can change:
 - `attatchment>` >> `attachment>`
 - `="games.strategy.triplea.attatchments` >> `="games.strategy.triplea.attachments`
 
-For a more defined change, instead of the aforementioned:
-
-- `Attatchment` >> `Attachment`
```

Comment:
Assuming that all changes are supposed to be case sensitive (something that used to be explicitly stated) changing "attatchment" to "attachment" is a different change from changing "Attatchment" to "Attachment", so what follows are not procedural alternatives to changing "attatchment" to "attachment", thus the information now given is wrong.

Reply:
The 2.x engine is actually becoming case insenstive. I think the assumption is that the changes are going to be case sensitive because the casing is specified.

Reply:
I would certainly advice against that, but this is off topic. Thanks for the information.